### PR TITLE
Added additional features to QA Module

### PR DESCRIPTION
**Pull request by yoko-8**
- Modal windows: the "Add an Answer" and "Ask Your Question" buttons now open modal windows (which were NOT made with a library).
- Hitting the "submit" button in these windows sends a post request to the API.

### CODE REVIEW

File: package-lock.json
Comment (davidguy3237) [inline package-lock.json:1]: It looks like this still has remnants of react-modal. See if you can get npm to create a new one without it included.

**Review by davidguy3237 — CHANGES_REQUESTED** [review]
Looks fine other than package-lock.json.
If you delete your package-lock.json and then rerun npm install. I believe it should remake the file again. Then just make a separate commit with just that file.

File: package-lock.json
Comment (yoko-8) [reply package-lock.json:1]: Got it. I'll get it remade

**Review by davidguy3237 — APPROVED** [review]

**Comment by yoko-8** [comment]
[Implement Add A Question button (opens modal window)](https://trello.com/c/34E1UecX/67-implement-add-a-question-button-opens-modal-window)

**Comment by yoko-8** [comment]
[Implement Add an Answer button (opens modal window)](https://trello.com/c/mEPtHkEA/68-implement-add-an-answer-button-opens-modal-window)

**Pull request merged**